org-docs/CHARTER.md

```diff
@@ -68,6 +68,14 @@ In order to be eligible to be a Organization project, a project must:
 
 Amendments to this charter, the [antitrust policy](./ANTITRUST.md), the [trademark policy](./TRADEMARKS.md), or the [code of conduct](./CODE-OF-CONDUCT.md) may only be made with at least a 3/4 affirmative vote of the Steering Committee.
 
+**9.1 Officers and Duties**. Every six months, the Steering Committee will conduct a vote for the following officers:
+
+* **Chair**. The Chair's main duty is to chair Steering Committee meetings. The Chair gives overall leadership to the Steering Committee, and should ensure that the Steering Committee members understand the roles and responsibilities, and debate its mission. The Chair also ensures that Steering Committee meetings are effectively run, and that decisions are made and properly recorded.
```

Comment:
"The Chair's main duty is to chair Steering Committee meetings" -> "The Chair's main duty is to run the Steering Committee meetings"